src/language-js/printer-estree.js

```diff
@@ -5285,22 +5285,79 @@ function isObjectType(n) {
 // eg; `describe("some string", (done) => {})`
 function isTestCall(n) {
   const unitTestRe = /^(skip|(f|x)?(it|describe|test))$/;
+
+  /*
+  * The below check makes sure that it('very long', async(()=>{//code}))
+  * async(()) doesn't break
```

Comment:
This comment seems outdated? or probably placed the wrong place.

And also `async(())` -> `async()` or `async(()=>{})`, `()=>{//code}` -> `()=>{/*code*/}`.